### PR TITLE
feat: import cibr data

### PR DESCRIPTION
**Pull request by XULU42**
1. register subject，array, task and dataset meta info for neucyber data;
2. add dataset config for neucyber data, which would be used in future;
3. add example pretrain config f32_pretrain.yaml
4. test for pretrain trainning process, check dataloader batch data data shape，dtype, trainning loss which are valid

### CODE REVIEW

**Review by azzhu — APPROVED** [review]
The code is beautifully written, due to limited capabilities, no problem has been found yet.
